server/packaging/ubuntu_xenial/kontena-server/DEBIAN/control

```diff
@@ -2,5 +2,5 @@ Package: kontena-server
 Version: VERSION
 Maintainer: info@kontena.io
 Architecture: all
-Depends: docker-engine (>= 1.10) | docker.io (>= 1.11) | docker-ce | docker-ee, resolvconf
```

Comment:
The `resolvconf` dependency accidentially slipped in here in #1173, it's not actually needed on the server...